src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
@pburkholder This:  https://technet.microsoft.com/en-us/library/security/cc750357.aspx#_Microsoft_FIPS_140  says that FIPS is not enforced by the OS, and it is up to each application to abide by the FIPS security policy